app/components/Reactions/ReactionPickerFooter.module.css

```
@@ -2,10 +2,9 @@
 
 .reactionPickerFooter {
   width: 100%;
-  height: 50px;
-  display: flex;
+  height: 60px;
   align-items: center;
-  padding: 15px;
+  padding: 5px 15px 0;
```

would be nice to use our variables to get the correct spacings
```suggestion
  padding: var(--spacing-sm) var(--spacing-md) 0;
```